### PR TITLE
Replace ToolbarButton to ActionButton (from SDK)

### PR DESCRIPTION
**Pull request by seleznev**
It should works better with new versions of Firefox (36+) and fix problem from #4.

Docs: https://developer.mozilla.org/en-US/Add-ons/SDK/Low-Level_APIs/ui_button_action#Badged_buttons

Screenshot:
screenshot from 2015-01-24 18 58 59

---

Please be careful, it doesn't work with Firefox 35!

If you need .xpi for testing, please tell me.

### CODE REVIEW

File: extension/lib/main.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const data = require('sdk/self').data;
 const Request = require('sdk/request').Request;
 const pageWorker = require('sdk/page-worker');
 const timers = require('sdk/timers');
-const toolbarButton = require('toolbarbutton/toolbarbutton').ToolbarButton;
+const { ActionButton } = require("sdk/ui/button/action");
```
Comment (sindresorhus) [inline extension/lib/main.js]: single-quotes

Comment (seleznev) [inline extension/lib/main.js]: Done. Sorry about that.

**Comment by sindresorhus** [comment]
You sir are awesome! :)

Glad to see Firefox finally added badge support to the action button.

> If you need .xpi for testing, please tell me.

Would be even better if you could publish it?
I've added you to the repo and the addon on the Firefox Addon Dev Hub.

**Comment by seleznev** [comment]
> Would be even better if you could publish it?

Yep. %)

Well, I will send it to AMO, but on week of February 17. The badge feature still can be removed from Beta builds...

But I will send "beta build" right now.

Many thanks!

**Comment by sindresorhus** [comment]
Thank you :)

**Pull request closed without merging**